build/jshint/xfail.js

```diff
@@ -1,34 +1,20 @@
-// custom jshint reporter, allows errors in files listed in the
```

Comment:
Sounds good. For now I will rename, but I will also look at changing this out in the future for another. We are switching over to eslint looks like, so it might just be time to move onto the next linter! Feels like the neverending linter migration :)